### ISSUE TITLE
As an end-user, I want to be able to bulk-import items into a viewer from a search results page

### ISSUE THREAD

**Issue by emmastanford** [IIIF/iiif-stories#86]
# Description

Using the search tool in Gallica, a user has found six manuscripts that they want to compare in Mirador. Instead of opening up a Mirador instance and adding each manifest separately, they'd like to do one of three things:

- automatically import all the manifests into a Mirador instance with the right number of slots. 
- import the manifests as a collection into Mirador.
- add the manifests to a customized version of the "Add item" page in Mirador, so that they can add or remove them as they like.

These manifests could be coming from a general search results list or from a user-created "collection" within a digital librar platform.

## Additional Background

[Frick Digital Collections](http://digitalcollections.frick.org/digico/#/) already allows users to add items to a personal collection and then open them in a right-sized Mirador container.

**Comment by azaroth42** [comment]
Considered in scope.  Could be implemented by sending a Collection of manifests to the viewer? Or six imports of individual manifests.
Exactly what the client does with the manifests is up to the client.

**Comment by benwbrum** [comment]
I agree with @azaroth42 -- rendering search results as a collection can work really well, as clients which can already consume collections don't have to write any new code.

**Comment by beaudet** [comment]
Is it documented anywhere which IIIF clients currently support collections or collections of collections?

**Comment by benwbrum** [comment]
At the moment, I think that Mirador and FromThePage are the only IIIF clients which can consume collections in some way.

**Comment by mattmcgrattan** [comment]
The Universal Viewer also supports collections, and will render them as part of the table of contents/index.

**Comment by jpstroop** [comment]
@benwbrum Does Mirador support IIIF collections? This has been a show stopper for us--I guess I should check in!
